packages/manager/modules/pci/src/projects/project/private-registry/list/list.controller.js

```diff
@@ -0,0 +1,91 @@
+export default class {
+  /* @ngInject */
+  constructor(
+    $state,
+    $stateParams,
+    CucCloudMessage,
+    CucControllerHelper,
+    CucServiceHelper,
+    privateRegistryService,
+    PRIVATE_REGISTRY_STATUS_MAP,
+    PRIVATE_REGISTRY_STATUS,
+  ) {
+    this.$state = $state;
+    this.$stateParams = $stateParams;
+    this.projectId = $stateParams.projectId;
+    this.CucCloudMessage = CucCloudMessage;
+    this.cucControllerHelper = CucControllerHelper;
+    this.cucServiceHelper = CucServiceHelper;
+    this.privateRegistryService = privateRegistryService;
+    this.PRIVATE_REGISTRY_STATUS_MAP = PRIVATE_REGISTRY_STATUS_MAP;
+    this.PRIVATE_REGISTRY_STATUS = PRIVATE_REGISTRY_STATUS;
+  }
+
+  $onInit() {
+    this.loadMessages();
+    this.getRegistryList(false);
+  }
+
+  loadMessages() {
+    this.CucCloudMessage.unSubscribe('pci.projects.project.private-registry.list');
+    this.messageHandler = this.CucCloudMessage.subscribe('pci.projects.project.private-registry.list', { onMessage: () => this.refreshMessages() });
+  }
+
+  refreshMessages() {
+    this.messages = this.messageHandler.getMessages();
+  }
+
+  getRegistryList(clearCache) {
```

Comment:
You should use state `resolves` for this (https://github.com/ovh-ux/manager/blob/master/packages/manager/modules/pci/src/projects/project/instances/instances.routing.js#L12)
All params resolved by the state are in cache and won't be resolved again unless if you redirect to the state with `reload` param set to `true`.

Reply:
This variable is sent to the `api-services` call to clearCache. Not variable passed between states.